lib/cmdlib/instance_migration.py

```diff
@@ -953,7 +953,10 @@ def _ExecMigration(self):
         raise errors.OpExecError("Could not migrate instance %s: %s" %
                                  (self.instance.name, msg))
 
-      if result.payload.status != constants.HV_MIGRATION_ACTIVE:
+      if ms.postcopy_status == "switched":
```

Comment:
`"switched"` should be a constant, as it is also used in hv_kvm. You could even use the already-defined `HV_KVM_MIGRATION_POSTCOPY_ACTIVE` constant, since the value of the `postcopy_status` field is opaque.